packages/block-library/src/template-part/edit/index.js

```diff
@@ -67,11 +69,52 @@ function ReplaceButton( {
 	);
 }
 
+function NonEditableTemplatePartPreview( {
```

Comment:
This preview component does not handle the case where a template part is missing